core/jobs/batch_jobs/exp_migration_jobs.py

```diff
@@ -233,6 +234,16 @@ def _update_exploration(
                     opportunity_services
                     .update_opportunity_with_updated_exploration(
                         migrated_exp.id))
+            exp_versions_diff = (
+                exp_domain.ExplorationVersionsDiff(list(exp_changes))
+            )
+            # Trigger statistics model update.
+            new_exp_stats = stats_services.get_stats_for_new_exp_version(
+                migrated_exp.id, migrated_exp.version,
+                list(migrated_exp.states.keys()),
+                exp_versions_diff, None)
+            # Puts new stats model corresponding to the new version.
+            stats_services.create_stats_model(new_exp_stats)
```

Comment:
Note: These lines actually puts models. But based on the existing structure of the job, we seem to return a list of models to be put and then put them all at once. So it seems a bit problematic to do this. I'll try to update this in a bit before testing.

Reply:
This has been updated now.